specs/InfoBadge/InfoBadge-spec.md

```diff
@@ -0,0 +1,526 @@
+
+## Using an InfoBadge in another control
+Use InfoBadge as you would any other control – simply add the InfoBadge markup where you’d like it to appear. Since InfoBadge inherits from `Control`, it has all the built-in positioning properties such as margin, alignment, padding, and more which you can take advantage of to position your InfoBadge exactly where you want it. Once the InfoBadge is positioned, use the `IsOpen` property to make the InfoBadge appear and disappear based on user actions, program logic, counters, etc. Note that the `IsOpen` property is set to `true` by default.
+
+If you’re placing an InfoBadge inside of another control, such as a Button or a ListViewItem, note that it is not recommended for the InfoBadge to be visible past the footprint of the parent control. So if your InfoBadge is inside of a button, for instance, it should not be visible past the corners of the Button’s overall footprint. 
+
+## Changing, hiding, and switching between types of InfoBadge
+To show or hide an InfoBadge, simply set the `InfoBadge.IsOpen` property to the appropriate value. On showing and hiding, the InfoBadge will animate.
+
+You can change the icon or number displayed in an InfoBadge while it is being shown. Decrementing or incrementing a numeric InfoBadge based on user action can be achieved by setting/manipulating the value of `InfoBadge.Value`. Changing the icon of an InfoBadge can be achieved by setting `InfoBadge.IconSource` to a new `IconSource` object. When changing icons, ensure that the new icon is the same size as the old icon to avoid a jarring visual effect.
+
+### Default behavior
+If neither `InfoBadge.Value` nor `InfoBadge.IconSource` are set, the InfoBadge defaults to showing a dot. On creation of the InfoBadge, both of these properties default to their sentinel values, hence the InfoBadge defaulting to show a dot. If both the `Value` and `IconSource` properties are set, the InfoBadge will honor the `Value` property and display a number value.  In order to hide an InfoBadge once any of its properties have been set, the `IsOpen` property will need to be set to `false`. 
+
+You can also change the InfoBadge’s type while it is being shown. To change the type of InfoBadge, be sure that the current type’s corresponding property (`Value` or `IconSource`) is set to its sentinel value (-1 or `null`), and set the new type’s property equal to an appropriate value. To change the type of InfoBadge from numeric or icon to a dot type InfoBadge, make sure that `InfoBadge.Value` is set to -1 and `InfoBadge.IconSource` is set to `null`. 
```

Comment:
I expect [`ClearValue`](https://docs.microsoft.com/uwp/api/windows.ui.xaml.dependencyobject.clearvalue) will work too?

Reply:
Good callout, that would definitely be easier than remembering sentinel values. 

As long as the sentinel values we're using for `Value` and `IconSource` are the same as the ones defined in the int and IconSource types themselves (I don't see why they wouldn't be), `ClearValue` should work.